ios/Converse/Supporting/Expo.plist

```diff
@@ -11,6 +11,6 @@
     <key>EXUpdatesURL</key>
     <string>https://u.expo.dev/49a65fae-3895-4487-8e8a-5bd8bee3a401</string>
     <key>EXUpdatesRuntimeVersion</key>
-    <string>1.3.10</string>
+    <string>2.0.7</string>
```

Comment:
🤷  dunno how this got changed

Reply:
Looks like expo prebuild was run, this change is not needed